src/SignalR/samples/ClientSample/RawSample.cs

```diff
@@ -34,10 +34,12 @@ public static async Task<int> ExecuteAsync(string baseUrl)
             baseUrl = string.IsNullOrEmpty(baseUrl) ? "http://localhost:5000/chat" : baseUrl;
 
             Console.WriteLine($"Connecting to {baseUrl}...");
-            var connection = new HttpConnection(new Uri(baseUrl));
+
+            var connection = new HttpConnection(new HttpConnectionOptions() { Url = new Uri(baseUrl) }, TransferFormat.Text, loggerFactory: null);
```

Comment:
Is this the simplest way to construct the HttpConnection now? Can we add some defaults (like the TransferFormat) so this sort of code doesn't break:

https://github.com/Azure/azure-signalr/blob/7257fda52f219601f8efc34dd240728a9c4ea31c/src/Microsoft.Azure.SignalR.Common/ServiceConnections/ConnectionFactory.cs#L32-L44